CMakeLists.txt

```diff
@@ -116,15 +118,17 @@ if("${CMAKE_EXECUTABLE_SUFFIX}" STREQUAL "")
   set(CMAKE_EXECUTABLE_SUFFIX ".x")
 endif()
 
+# Create a custom target to allow preparing for style checks
```

Comment:
how come we're moving this above the python config? we just don't need it?

Reply:
Oh, I just moved it above the big block comment `Morpheus Python Setup` so non-python stuff is before and python stuff is after. It doesnt matter the order of this an `morpheus_utils_python_configure()`